src/cms-content/learn/utils.ts

```diff
@@ -0,0 +1,5 @@
+import { deburr, words } from 'lodash';
+
+export function sanitizeID(sectionId: string): string {
+  return deburr(words(sectionId).join('-')).toLowerCase();
```

Comment:
nice! i imagine `deburr` is intended for when we include the spanish translations?

Reply:
it takes out any accents or other non-ascii characters (just in case)